sensors/depth/depth.go

```diff
@@ -0,0 +1,53 @@
+// Package depth implements the Depth sensor
+package depth
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/registry"
+
+	"go.viam.com/slam/config"
+	"go.viam.com/slam/sensors"
+	"go.viam.com/slam/sensors/utils"
+)
+
+// Depth represents a depth sensor.
+type Depth struct {
+	Name         string
+	depth        camera.Camera
+	DataRateMsec int
```

Comment:
Why is this part of the sensor, rather than the data process?

Reply:
That's a good point. I designed this with an older version of the movement scope doc in mind. I will remove this from the sensor and leave the ownership with the data process.